### DISCUSSION TITLE
[Feature or bug] albums should be "shared" if they are shared

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6127] [Feature Request]
### The feature

Currently, albums can be "albums" or "shared albums".
Albums should be "shared albums" as soon, as they are really shared. Or there should be at least a switch in the GUI to make them "shared albums". 

I have some albums, which are shared with other users, but they are not shown as "shared albums" and they are not listed when I want to add content to shared albums. Is it a bug or is it by design?

I don't understand, why the classification "not shared album" vs "shared album" doesn't match automatically: if they are really shared or not. Why not just automate the classification? If an album will be shared with someone, then it should be switched from "album" to "shared album". If all sharing is removed, it should be switch back. In General, I don't understand, why a user is asked to create a normal or a shared album. He should just create albums, and later he can share or not.

I can't find in the database a flag, if it is shared or not. Maybe this is detected automatically, and the missing detection is just a bug?

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by NicholasFlamy** [comment]
I just checked, Google Photos shows the album as shared and shows it in the sharing screen as soon as you share it.

**Reply by NicholasFlamy** [reply]
I just checked, immich functions just like Google Photos in this regard. (Minus having the album say shared as seen in this screenshot)
![Screenshot_20240129_075526](https://github.com/immich-app/immich/assets/30300649/776098d4-d44c-4d6f-86a8-2018fd9dc7d0)
![Screenshot_20240129_075623_Photos](https://github.com/immich-app/immich/assets/30300649/3cf6fd4e-e27c-4c9e-ab10-dce9f8946972)
